LSP-intelephense.sublime-settings

```diff
@@ -14,8 +14,8 @@
 	],
 	"initializationOptions": {
 		"clearCache": false,
-		// "storagePath": "${temp_dir}/intelephense",
-		// "globalStoragePath": "${home}/.intelephense",
```

Comment:
In my experiment if there is no specified `globalStoragePath`, the default `globalStoragePath` which intelephense uses doesn't have that prefixed dot.